pkgs/desktops/lxqt/optional/screengrab/build-date-time.patch

```diff
@@ -0,0 +1,22 @@
+Description: Build date time
+Author: Alf Gaida <agaida@siduction.org>
+
+--- a/src/core/ui/about.cpp
++++ b/src/core/ui/about.cpp
+@@ -31,16 +31,8 @@ AboutDialog::AboutDialog(QWidget *parent):
+     _ui->setupUi(this);
+     _ui->labAppName->setText(_ui->labAppName->text() + QString(" <b>") + qApp->applicationVersion() + QString("</b>"));
+ 
+-    QString versionInfo;
+-    versionInfo = tr("built on ");
+-    versionInfo.append(__DATE__);
+-    versionInfo.append(" ");
+-    versionInfo.append(__TIME__);
```

Comment:
There modifications in nixos which replaces __DATE__ and __TIME__ by deterministic values. So this is not required.